molecule/ci/molecule.yml

```diff
@@ -0,0 +1,26 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: grsec_build_trusty
+    image: debian:jessie
```

Comment:
Inconsistent here—we're calling it trusty, but actually building against Jessie. Since CI is currently passing, I'm going to change the platform _name_, rather than switch the image. We can expand to more platforms later, given how tiny the CI config is.